README.md

```diff
@@ -56,7 +56,9 @@ Technology - Lately I prefer to use React with Redux and webpack, but I'm also f
 * Can you describe the difference between progressive enhancement and graceful degradation?
 * How would you optimize a website's assets/resources?
 * How many resources will a browser download from a given domain at a time?
-  * What are the exceptions?
+  * What are the exceptions?  
```

Comment:
jest ok, tylko nie znalazłem nic o exceptions (to jest imo jedno pytanie rozbite na dwie linijki)